frontend/src/framework/components/ParameterListFilter/private-utils/smartNodeSelectorUtils.ts

```diff
@@ -0,0 +1,179 @@
+import { Parameter, ParameterIdent, ParameterType } from "@framework/EnsembleParameters";
+import { TreeDataNode } from "@lib/components/SmartNodeSelector";
+
+export const ParameterParentNodeNames = {
+    NAME: "Name",
+    GROUP: "Group",
+    CONTINUOUS: "Continuous", // For Parameter.type === ParameterType.CONTINUOUS
+    DISCRETE: "Discrete", // For Parameter.type === ParameterType.DISCRETE
+    IS_CONSTANT: "Constant", // For Parameter.isConstant === true
+    IS_NONCONSTANT: "Nonconstant", // For Parameter.isConstant === false
+    IS_LOGARITHMIC: "Logarithmic", // For Parameter.isLogarithmic === true
+    IS_LINEAR: "Linear", // For Parameter.isLogarithmic === false
+};
+
+export function fromParameterTypeToNodeName(type: ParameterType): string {
```

Comment:
Consider renaming the function to use a verb, maybe `getNodeNameFromParameterType`?

Reply:
👍